src/sqlfluff/core/default_config.cfg

```diff
@@ -27,6 +27,11 @@ disable_noqa = False
 # Comma separated list of file extensions to lint
 # NB: This config will only apply in the root folder
 sql_file_exts = .sql,.sql.j2,.dml,.ddl
+# By default, files won't be fixed if they contain parse errors (even if those
+# errors are ignored with "noqa"). This option allows overriding that behavior:
+# if set to True, fixes *will* be applied. USE WITH CAUTION: MAY APPLY SPURIOUS
+# OR UNNECESSARY FIXES. IT MAY BREAK OR CORRUPT YOUR SQL.
+fix_even_unparsable = False
```

Comment:
Could we make this smaller?

```suggestion
# Allow fix to run on files, even if they contain parsing errors
# Note altering this is NOT RECOMMENDED as can corrupt SQL
fix_even_unparsable = False
```

Reply:
I often wonder how much we need to scare people. We'll go with the shorter phrasing for now. We can make it scarier later if people keep shooting themselves in the foot and reporting issues. 😅

Reply:
I was thinking the big block made it stand out (which we don’t really want)! 😁 So tried to minimise it a bit.